packages/vm/test/t8n/README.md

```diff
@@ -0,0 +1,40 @@
+# EVM T8NTool
+
+T8NTool, or Transition Tool, is a tool used to "fill tests" by test runners, for instance <https://github.com/ethereum/execution-spec-tests/>. These files take an input allocation (the pre-state), which contains the accounts (their balances, nonces, storage and code). It also provides an environment, which holds relevant data as current timestamp, previous block hashes, current gas limit, etc. Finally, it also provides a transactions file, which are the transactions to run on top of this pre-state and environment. It outputs the post-state and relevant other artifacts, such as tx receipts and their logs. Test fillers will take this output to generate relevant tests, such as Blockchain tests or State tests, which can then be directly ran in other clients, or using EthereumJS `npm run test:blockchain` or `npm run test:state`.
+
+## Using T8Ntool to fill `execution-spec-tests`
+
+To fill `execution-spec-tests` (or write own tests, and test those against the monorepo), follow these steps:
+
+1. Clone <https://github.com/ethereum/execution-spec-tests/>.
+2. Follow the installation steps: <https://github.com/ethereum/execution-spec-tests?tab=readme-ov-file#quick-start>.
+
+To fill tests, such as the EIP-1153 TSTORE/TLOAD tests, run:
+
+- `fill -vv -x --fork Cancun tests/cancun/eip1153_tstore/ --evm-bin=../ethereumjs-monorepo/packages/vm/test/t8n/ethereumjs-t8ntool.sh`
```

Comment:
Should this command work correctly?  I get validation errors from the CLI when I run this particular test and it's not obvious to me if I mangled something trying to install the execution tests or if this is a flaw on our side?

Reply:
Disregard.  This was user error.